src/main/java/crazypants/enderio/conduit/item/filter/ItemFilter.java

```diff
@@ -93,43 +90,60 @@ public boolean doesItemPassFilter(NetworkedInventory inv, ItemStack item) {
     return doesItemPassFilter(item);
   }
 
+  @SuppressWarnings("SimplifiableConditionalExpression")
   public boolean doesItemPassFilter(ItemStack item) {
-    if(!isValid()) {
-      return true;
+    boolean allow = isValid();
+
+    if(allow) {
+      allow = itemMatched(item);
     }
-    boolean matched = itemMatched(item);
-    return isBlacklist ? !matched : matched;
+
+    return isBlacklist ? !allow : allow;
   }
 
   private boolean itemMatched(ItemStack item) {
     if(item == null) {
       return false;
     }
-    boolean doFuzzy = false;
-    boolean fuzzyValue = false;
-    if(fuzzyMode != FuzzyMode.DISABLED && item.getItem().isDamageable()) {
-      doFuzzy = true;
-      fuzzyValue = fuzzyMode.compare(item);
+
+    boolean checkDamage = fuzzyMode != FuzzyMode.DISABLED;
+    boolean damageMatched = false;
```

Comment:
These two variables could probably be collapsed into one, e.g. `fuzzyOk`.

Reply:
Few lines below:
```
if(checkDamage && !damageMatched) {
   matched = false;
}
```

So I tried to merge it into one variable too, but returned it back because of that expression.